packages/playground/assets/__tests__/assets.spec.ts

```diff
@@ -57,7 +70,10 @@ describe('raw references from /public', () => {
   })
 })
 
-describe('asset imports from js', () => {
+describe('asset imports from js', (test) => {
+  test.before(async () => await uvuSetup(__dirname))
+  test.after(uvuReset)
```

Comment:
these 2 lines repeat a lot. can this be refactored into a describe wrapper or some other utility so that tests don't have to add them every time?

Reply:
Yes, I am still not sure to put in the describe wrapper, so I repeat at least for now.